docker/php-build-stepup/Dockerfile

```diff
@@ -20,11 +17,17 @@ ENV NVM_DIR /usr/local/nvm
 ENV NODE_VERSION 10
 
 # Install nvm with node and npm
-RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash
```

Comment:
Alltough this syntax works, it's more common to put ```&& \``` at the end of the line and start the newline with the next command